competition/track1/train/train/info.py

```diff
@@ -28,6 +28,6 @@ def step(
         obs, reward, done, info = self.env.step(action)
 
         for agent_id in info.keys():
-            info[agent_id]["is_success"] = bool(info[agent_id]["score"])
+            info[agent_id]["is_success"] = bool(info[agent_id].get("score", True))
```

Comment:
The `is_success` is set here to enable the SB3 library to count the number of successfully completed episodes and to print it during evaluation. An episode is considered to be successfully completed if the agent reaches the goal, i.e., `obs.events.reached_goal=True`.

Why is the `score` being set to `True` by default here, as this is misleading?